app/views/teams/show.html.erb

```diff
@@ -3,6 +3,52 @@
     <h1><%= i @team.team_type.icon %> <%= @team.identifier_name %></h1>
   </div>
   <div class="card-body">
+    <div class="row">
+      <div class="col">
+        <div style="margin-bottom: 20px;">
+          <h2>People</h2>
```

Comment:
The people list is going to be huge in some cases. I'd recommend doing this more like the `people/index` view with cards for everyone underneath the main team card.